packages/babel-plugin-transform-runtime/src/index.js

```diff
@@ -202,6 +221,11 @@ export default declare((api, options, dirname) => {
       moduleName,
       path.resolve(dirname, absoluteRuntime === true ? "." : absoluteRuntime),
     );
+  } else if (relativeRuntime !== false) {
+    modulePath = resolveRelativeRuntime(
+      moduleName,
+      path.resolve(dirname, relativeRuntime === true ? "." : relativeRuntime),
```

Comment:
This doesn't match my expectations. Wouldn't we need to calculate this path relative to the JS file being compiled, rather than using the directory?

I also wanted this to strictly be a boolean option because I think of the flags this way:

- `absoluteRuntime` 
    - `false` to just inject `@babel/runtime/<whatever>` 
    - `true` to resolve that to a specific path first
- `useRelativeRuntime`
    - only allowed when `absoluteRuntime` is `true` or a `string`
    - `false` - current behavior
    - `true` - take the resolved path from before and resolve it relative to the JS file it is being inserted into, instead of inserting an absolute path.

If I were doing this from scratch, these might have been nicer as
```
type Opts = {
  resolveRuntime?: boolean | string,
  resolveRelativePath?: boolean,
};
```
or
```
type Opts = {
  resolve?: boolean | string | {
    dirname?: string,
    relative?: boolean,
  },
};
```
or something